input/input_selection.txt

```diff
@@ -26,7 +26,6 @@ veto
 	IsoPionTrackVeto
 vetop
```

Comment:
i would just remove vetop entirely and replace it with veto (since they are now identical)

Reply:
Done